### ISSUE TITLE
Table is showing wrong data

### ISSUE THREAD

**Issue by mnuessler** [hypebeast/bundesliga-cli#2]
The Bundesliga table that is displayed has completely wrong data. The number of matches seems to be correct, but rank, wins, losses and points are all wrong for some reason. Using version 0.2.0.

```bash
$ buli table
+------+--------------------------+---------+------+-------+--------+-------+----+--------+
| Rank | Club                     | Matches | Wins | Draws | Losses | Goals | GD | Points |
+------+--------------------------+---------+------+-------+--------+-------+----+--------+
|    1 | Borussia Dortmund        |      11 |    7 |     1 |      3 |  14:6 |  8 |     22 |
|    2 | Bayern München           |      11 |    6 |     3 |      2 |  9:4  |  5 |     21 |
|    3 | VfL Wolfsburg            |      11 |    5 |     5 |      1 |  8:4  |  4 |     20 |
|    4 | SV Darmstadt 98          |      10 |    5 |     3 |      2 |  7:4  |  3 |     18 |
|    5 | 1. FSV Mainz 05          |      11 |    4 |     5 |      2 |  9:7  |  2 |     17 |
|    6 | Hertha BSC               |      11 |    4 |     3 |      4 |  6:7  | -1 |     15 |
|    7 | Eintracht Frankfurt      |      11 |    3 |     6 |      2 |  9:6  |  3 |     15 |
|    8 | Borussia Mönchengladbach |      11 |    3 |     5 |      3 | 11:10 |  1 |     14 |
|    9 | TSG 1899 Hoffenheim      |      11 |    2 |     7 |      2 |  7:7  |  0 |     13 |
|   10 | FC Augsburg              |      11 |    3 |     4 |      4 |  7:13 | -6 |     13 |
|   11 | Bayer 04 Leverkusen      |      11 |    2 |     6 |      3 |  5:6  | -1 |     12 |
|   12 | FC Ingolstadt 04         |      11 |    1 |     9 |      1 |  2:2  |  0 |     12 |
|   13 | 1. FC Köln               |      11 |    2 |     5 |      4 |  4:9  | -5 |     11 |
|   14 | Werder Bremen            |      11 |    2 |     5 |      4 |  8:8  |  0 |     11 |
|   15 | FC Schalke 04            |      11 |    2 |     5 |      4 |  4:6  | -2 |     11 |
|   16 | VfB Stuttgart            |      10 |    2 |     4 |      4 |  7:9  | -2 |     10 |
|   17 | Hamburger SV             |      10 |    1 |     6 |      3 |  3:4  | -1 |      9 |
|   18 | Hannover 96              |      10 |    1 |     2 |      7 |  3:11 | -8 |      5 |
+------+--------------------------+---------+------+-------+--------+-------+----+--------+
```

**Comment by hypebeast** [comment]
The reason is that Openligadb has changed their API. I'll take care of that.

Thanks for your issue.

**Comment by hypebeast** [comment]
This bug should be fixed in version 0.2.1.

**Comment by mnuessler** [comment]
Thanks, it's working correctly now.
